bot/exts/fun/latex.py

```diff
@@ -110,12 +110,15 @@ async def latex(self, ctx: commands.Context, *, query: str) -> None:
                     with open(image_path, "wb") as out_file:
                         await self._generate_image(TEMPLATE.substitute(text=query), out_file)
                 except InvalidLatexError as err:
-                    logs_paste_url = await self._upload_to_pastebin(err.logs)
                     embed = discord.Embed(title="Failed to render input.")
-                    if logs_paste_url:
-                        embed.description = f"[View Logs]({logs_paste_url})"
+                    if err.logs is None:
+                        embed.description = "No logs available."
                     else:
-                        embed.description = "Couldn't upload logs."
+                        logs_paste_url = await self._upload_to_pastebin(err.logs)
+                        if logs_paste_url:
+                            embed.description = f"[View Logs]({logs_paste_url})"
+                        else:
+                            embed.description = "Couldn't upload logs."
```

Comment:
I think this should raise a warning

Reply:
I disagree. If the logs had too many characters and couldn't be uploaded to pastebin, then the core dev team don't need to be told by raising a warning and a sentry issue.

If it fails for another reason, then sure. At the moment however there is no way to tell.

I think we should lower this to an info at least. Maybe in a subsequent PR we could try to truncate logs before uploading to the paste service, this would mean an error being raised would be something that the core dev team would care about.

Reply:
Right. I didn't realize it would also return None if the text is too long. 

Although, I'd argue that there should be a character limit that is being checked by the log, and we should either have another description message or truncate the logs.

Reply:
Yea, I agree on the last point, and is what I mentioned in the last paragraph of my last comment.

For now I'd say revert the log change and then we can raise an issue to do the truncation.